client/js/app.modules/menu/menu.controller.js

```diff
@@ -0,0 +1,6 @@
+/**
+ * @ngInject
+ * */
+module.exports = function ($scope, $http, GlobalData){
+    this.GlobalData = GlobalData;
```

Comment:
Same: Use lowercase for properties ("globalData"). But its better to show only particular methods on scope (not whole service).
Maybe even take a look into using pubsub messages to hide/show menu?